.github/workflows/ci-docker-hotrod.yml

```diff
@@ -18,6 +18,10 @@ permissions:
 jobs:
   hotrod:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        jaeger-version: [v1, v2]
```

Comment:
should we specify binary here instead?
as `examples/hotrod/docker-compose-v2.yml` depends on `jaeger` (v2) image and `examples/hotrod/docker-compose.yml` on `all-in-one` (v1)